src/Parser/Hunter/Shared/Modules/Features/FocusChart/FocusTracker.js

```diff
@@ -134,6 +137,10 @@ class FocusTracker extends Analyzer {
     this.lastEventTimestamp = eventTimestamp;
   }
 
+  get averageFocusGen() {
+    return this.totalFocusGenModifier / this.totalFocusGenModifiedTimes;
```

Comment:
I'm not entirely sure I understand everything going on in this class but wont this be off since you usually don't spent the same amount of time at each different modifier? I'd suggest you instead save the timestamp everytime the modifier is changed, and then do `this.totalFocusGenModifier += this.focusGen * (currentTimeStamp - lastTimeStamp)` to add the focus gen for that window.
Then you can do `return this.totalFocusGenModifier / this.owner.fightDuration;` to get the average focusGen